huaweicloud/common/validations.go

```diff
@@ -0,0 +1,147 @@
+package common
+
+import (
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func StandardVerify(min, max int) schema.SchemaValidateFunc {
```

Comment:
1. Some function names are too long, can they be shortened?
2. The function name should be intuitive so that the simplest code can be used to complete the call.

Reply:
There are too many types of characters to be checked.It is a simple way to explain in the function name.